CHICore/Classes/Utils/String/String+Height.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  String+Height.swift
+//  CHICore
+//
+//  Created by McSims on 04/04/2019.
+//
+
+import Foundation
+
+extension String {
+
+    public func height(withConstrainedWidth width: CGFloat, font: UIFont) -> CGFloat {
+        let constraintRect = CGSize(width: width, height: .greatestFiniteMagnitude)
+        let boundingBox = self.boundingRect(with: constraintRect, options: .usesLineFragmentOrigin, attributes: [NSAttributedString.Key.font: font], context: nil)
+        return boundingBox.height
+    }
+
+    public func width(withConstraintedHeight height: CGFloat, font: UIFont) -> CGFloat {
+        let constraintRect = CGSize(width: .greatestFiniteMagnitude, height: height)
+        let boundingBox = self.boundingRect(with: constraintRect, options: .usesLineFragmentOrigin, attributes: [NSAttributedString.Key.font: font], context: nil)
+        return boundingBox.width + 25
```

Comment:
will '25' be always needed for this ? might not be the best idea to leave it here

Reply:
totally agree

Reply:
removed